00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,542 @@
+
+- it has two or more approving reviews
+- every reviewer that has submitted a review has updated their review to
+  "Approved"
+
+The latter exists to ensure that all feedback from all reviewers is addressed
+before merge.
+
+With concrete criteria in place, automated merging, a longer-term automation
+goal, may become feasible. This is represented by the final green arrow from
+`approved` to `merged` in the flowchart.
+
+## Drawbacks
+
+- The team will need to attend a few more meetings. The daily standups, as
+  currently designed, lose their focus on individual areas of the project per
+  day; this was useful in case an external contributor was interested in one
+  specific area of the project. However, we hope the added free-form discussion
+  time will allow for relevant questions to be asked and talked about anyway.
+
+- Implementing some of the automation exactly as specified may end up being a
+  challenge. There's further discussion of this below.
+
+## Rationale and alternatives
+
+There are many ways to approach sprints, and we can't claim this is the best
+possible sprint methodology out there; only time will tell if this works for us
+once we start using it. Additionally, this process is not set in stone; anything
+can change from one sprint to the next. However, there are a few nice
+characteristics worth highlighting:
+
+- Enarx is still a fairly small team, and we like some flexibility in our
+  process. The modular design of our board supports that -- for example, if we
+  consider a new issue or pull request particularly important, we can manually
+  assign it and have it skip the backlog entirely.
+- Moving to a single, organization-level board allows visitors to see a snapshot
+  of what's currently a top priority for Enarx.
+- With automation, this process has potential to minimize the busywork that
+  project developers have to deal with, and only require human judgment when it
+  is needed.
+- Running through a triage every day ensures issues are promptly reviewed by the
+  team after they are submitted.
+
+## Prior art
+
+The Enarx project's current process of topic-specific boards largely informs
+this proposal's approach to the backlog. In fact, this proposal could be
+considered an extension or adaptation of the current process in some ways.
+
+And, of course, the agile sprint methodology is widely employed just about
+everywhere in software; see
+[Atlassian](https://www.atlassian.com/agile/scrum/sprints) and
+[Trello](https://blog.trello.com/beginners-guide-scrum-and-agile-project-management)
+for good examples. And, of course, Red Hat
+[itself](https://developers.redhat.com/blog/2017/06/29/agile-software-development-the-red-hat-way/)
+uses it.
+
+## Unresolved questions
+
+The most significant unresolved question does not involve process, but rather
+the automation we hope to pair with it: whether the project board will
+live inside the main Enarx repository, or as an all-encompassing Enarx
+organization board.
```

Comment:
Isn't this question resolved now? I see under "Project Board" it claims that the board will be organization-level.

Reply:
You are right. I've updated it locally and will push it up soon.